Assets/MixedRealityToolkit-SDK/Features/Diagnostics/MixedRealityDiagnosticsManager.cs

```diff
@@ -31,7 +31,7 @@ private void InitializeInternal()
 
             if (Visible)
             {
-                GetDiagnosticVisualization();
+                DiagnosticVisualization.SetActive(Visible);
```

Comment:
`DiagnosticVisualization?.SetActive(Visible);`?

Reply:
Shouldn't be needed, although it's not clearly outlined why. If visible is true then the getter will always create if it's null. I'll add a comment to clarify the expectation in behavior.

Reply:
A few lines below you were also checking null ability. might want to remove that one then as well?

Reply:
This whole if block can be removed. The visibility property will correctly handle setting active, no need to do it again.